dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -545,29 +545,6 @@
         }
       ]
     },
-    "PreBuiltResult": {
-      "type": "object",
```

Comment:
This is also missing in common.json. I think we should add `type: object` there as well. In practice, it shouldn't make a difference. I think all our modern code generators don't really care, but we should compare generated outputs for any differences.